### PR TITLE
fix formatting issues

### PR DESCRIPTION
**Pull request by mairaw**
While reviewing the issue #3623, I've noticed that the topic (https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/how-to-localize-an-application) has some other issues:

- A broken table:
image

- Broken snippet references
image

While we wait for the team to confirm the new location of the samples, I thought I'd fix these other issues.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/how-to-localize-an-application?branch=pr-en-us-3625)

### CODE REVIEW

File: docs/framework/wpf/advanced/how-to-localize-an-application.md [hunk 1]
```diff
@@ -133,11 +127,11 @@ This tutorial explains how to create a localized application by using the LocBam
...
-    ||  
-    |-|  
-    |HelloApp.g.en-US.resources:window1.baml,Stack1:System.Windows.Controls.StackPanel.$Content,Ignore,FALSE, FALSE,,#Text1;#Text2;|  
-    |HelloApp.g.en-US.resources:window1.baml,Text1:System.Windows.Controls.TextBlock.$Content,None,TRUE, TRUE,,Hello World|  
-    |HelloApp.g.en-US.resources:window1.baml,Text2:System.Windows.Controls.TextBlock.$Content,None,TRUE, TRUE,,Goodbye World|  
+   ||  
+   |-|  
+   |HelloApp.g.en-US.resources:window1.baml,Stack1:System.Windows.Controls.StackPanel.$Content,Ignore,FALSE, FALSE,,#Text1;#Text2;|  
+   |HelloApp.g.en-US.resources:window1.baml,Text1:System.Windows.Controls.TextBlock.$Content,None,TRUE, TRUE,,Hello World|  
+   |HelloApp.g.en-US.resources:window1.baml,Text2:System.Windows.Controls.TextBlock.$Content,None,TRUE, TRUE,,Goodbye World|
```
Comment (JRAlexander) [inline docs/framework/wpf/advanced/how-to-localize-an-application.md]: There are formatting issues.  The table still appears to be broken.

**Review by JRAlexander — CHANGES_REQUESTED** [review]

**Comment by mairaw** [comment]
Thanks for checking @JRAlexander and adding the preview URL. Let's see if this change fixes the issue. It did on the Gauntlet preview at least.

**Comment by mairaw** [comment]
@JRAlexander it seems that my last commit fixed the issue. Can you give me your seal of approval? 😄

**Comment by JRAlexander** [comment]
Looks wonderful @mairaw!!!

**Comment by JRAlexander** [comment]
Looks wonderful @mairaw!!!

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**